explainer.md

```diff
@@ -104,10 +108,17 @@ Parameters:
 
 A typed field definition is an `object` definining the characteristics of a `Struct`'s typed field. It has the following members:
  - `type` - A [type definition](#type-definition), specifying the field's type.
- - `structural` [optional] - A `boolean` indicating that, if `true`, the type check performed when writing to the field is structural, not nominal.
  - `name` [optional] - A `string` or `symbol` used as an optional name for the field. If given, an accessor is created that allows reading and, if the field is writable, writing the field using a name in addition to its index.
  - `readonly` [optional] - A `boolean`. If `true`, the field can only be set via the type's constructor and is immutable afterwards.
 
+#### Struct type references
```

Comment:
Technically, these references can refer to both struct, array (and in the future, function) definitions, so maybe just call them "Reference value types"?

Reply:
We currently don't have array or function type definitions. I'd prefer to only make this language general once we can actually make use of that generality—otherwise I fear it might just be confusing. In particular, I think the first paragraph would have to be quite a bit more complex, without a clear reason for why.

I added a paragraph explaining that additional reference value types are possible.